docs/faq.rst

```diff
@@ -13,7 +13,7 @@ Yes, the `deduplication`_ technique used by
 Also, we have optional simple sparse file support for extract.
 
 If you use non-snapshotting backup tools like Borg to back up virtual machines,
-then these should be turned off for doing so. Backing up live VMs this way can (and will)
+then the VMs should be turned off before doing so. Backing up live VMs can (and will)
```

Comment:
Strictly taken one could also shut down, snapshot, turn on, make a backup of the snapshot while machine is already running again, remove snapshot. But without snapshot, the VM should stay off.

Yes, "turned off for the duration of the backup" is clearest, guess we should take that.

About development: you're welcome, we enjoy it. :) If you have old attic archives in the (converted) repo, make sure you use latest borg release 1.0.9 (see changelog).